projects/js-packages/publicize-components/src/components/media-section/index.js

```diff
@@ -110,14 +147,43 @@ export default function MediaSection() {
 						<MediaUpload
 							title={ REPLACE_MEDIA_LABEL }
 							onSelect={ onUpdateMedia }
-							allowedTypes={ ALLOWED_MEDIA_TYPES }
+							allowedTypes={ allowedMediaTypes }
 							render={ replaceMediaRender }
 						/>
 						<Button onClick={ onRemoveMedia } variant="link" isDestructive>
 							{ REMOVE_MEDIA_LABEL }
 						</Button>
 					</>
 				) }
+				{ validationError && (
+					<Notice
+						className={ styles.notice }
+						isDismissible={ true }
+						onDismiss={ onDismissClick }
```

Comment:
Should this be dismissable? Wouldn't you end up with an invalid image but no visible validation error?

Reply:
We can make it non-dismissable, but I think you cannot get to a state with this, where there is an invalid image selected, because if the image is invalid it won't be selected, or it will get removed if you get there by changing the connections. What do you think?

Reply:
Ah yeah I wasn't thinking about it correctly—the media never gets added if the notice appears. This is good then!